libminifi/src/processors/GetFile.cpp

```diff
@@ -220,15 +219,14 @@ void GetFile::pollListing(std::queue<std::string> &list, const GetFileRequest &r
   std::lock_guard<std::mutex> lock(mutex_);
 
   while (!_dirList.empty() && (request.batchSize == 0 || list.size() < request.batchSize)) {
-    std::string fileName = _dirList.front();
+    list.push(_dirList.front());
     _dirList.pop();
-    list.push(fileName);
   }
-
-  return;
 }
 
 bool GetFile::acceptFile(std::string fullName, std::string name, const GetFileRequest &request) {
+  logger_->log_debug("Checking file: %s", fullName);
```

Comment:
This might happen a lot? Almost useful to trace the execution. Log_trace maybe?

Reply:
Makes sense, changed to trace, thanks!